core/src/header.rs

```diff
@@ -557,6 +557,12 @@ impl Tag {
     }
 }
 
+impl fmt::Debug for Tag {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        write!(f, "Tag(0x{:04X}, 0x{:04X})", self.0, self.1)
```

Comment:
We can take advantage of pretty-printing (`#`) to add the prefixes:

```suggestion
        write!(f, "Tag({:#06X?}, {:#06X?})", self.0, self.1)
```